src/Form/Input.js

```diff
@@ -52,17 +52,27 @@ const StyledInput = createComponent({
 
 const StyledTextArea = StyledInput.withComponent('textarea');
 
+const validateValueProp = (props, propName, componentName) => {
+  if (props.type === 'number' && typeof props.value !== 'number') {
+    return new Error(`Invalid prop ${propName} supplied to ${componentName} with type="number", expected Number`);
+  }
+  if (typeof props[propName] !== 'string' && props.type !== 'number') {
```

Comment:
is there a case where you have to add the second check for `props.type !== 'number'`?

Reply:
Without that this would fail as invalid `<Input value={123} type="number" />`

Reply:
whoops, good point